tests/conftest.py

```diff
@@ -5130,3 +5136,1464 @@ def populated_profiler_store(
     profiler_store = empty_profiler_store
     profiler_store.set(key=profiler_key, value=profiler_config_with_placeholder_args)
     return profiler_store
+
+
+@pytest.fixture
+def alice_columnar_table_single_batch_context(
+    monkeypatch,
+    empty_data_context,
+    alice_columnar_table_single_batch,
+):
+    skip_if_python_below_minimum_version()
```

Comment:
Integration-test fixtures have been moved to `tests/conftest.py` eliminating the need for additional imports